samples/appendingEvents.go

```diff
@@ -0,0 +1,142 @@
+package samples
+
+import (
+	"context"
+	"log"
+
+	"github.com/EventStore/EventStore-Client-Go/client"
+	"github.com/EventStore/EventStore-Client-Go/messages"
+	"github.com/EventStore/EventStore-Client-Go/options"
+	"github.com/EventStore/EventStore-Client-Go/stream_position"
+	"github.com/EventStore/EventStore-Client-Go/streamrevision"
+)
+
+type TestEvent struct {
+	id            string
+	importantData string
+}
+
+func AppendToStream(db *client.Client) {
+	// region append-to-stream
+	data := TestEvent{
+		id:            "1",
+		importantData: "some value",
+	}
+
+	event, err := messages.NewJsonProposedEvent("some-event", data)
+
+	if err != nil {
+		panic(err)
+	}
+
+	options := options.AppendToStreamOptionsDefault()
+
+	result, err := db.AppendToStream(context.Background(), "some-stream", &options, event)
+	// endregion append-to-stream
+
+	log.Printf("Result: %v", result)
+}
+
+func AppendWithSameId(db *client.Client) {
```

Comment:
Maybe `AppendWithDuplicateID` will be better? I was thinking that it's the same ID as in `AppendToStream` at the beginning :sweat_smile:

Reply:
It's a sample files, this function name will never show up in the documentation website :smile:

Reply:
fair enough :ok: